spec/models/bale_spec.rb

```diff
@@ -27,6 +28,11 @@
         bale.organization = nil
         expect(bale).not_to be_valid
       end
+
+      it 'does not let create new ones without user' do
+        bale.user = nil
+        expect(bale).not_to be_valid
+      end
     end
```

Comment:
We must have a test that checks the serializer.

Reply:
It's right down there...